### ISSUE TITLE
4x2 Widget with hourly forecast

### ISSUE THREAD

**Issue by str1k3r** [deno750/Overdrop-Issues#1]
Please add subj.

Thanks.

**Comment by ianmacd** [comment]
Indeed. None of the widgets supports an hourly forecast. This makes the widgets much less useful than they could be.

For example, the temperature is set to rise by 10°C in Amsterdam today, placing the temperature above freezing. That's important to know, but all the widget shows me is the current, very cold temperature.

**Comment by str1k3r** [comment]
@ianmacd can't agree.

check my screenshot (weather app name is Weather Timeline):
![image](https://user-images.githubusercontent.com/19888/51746366-efc03a80-20b6-11e9-9f1a-c25c4c6d1756.png)

of course, current conditions are not the best to show the difference but still can inform you that later today you can have completely different forecast (sun -> rain, etc).

thanks.

**Comment by ianmacd** [comment]
I think you misunderstood me.

I thought you were saying that you think an hourly widget is a **good** idea. I was **agreeing** with you.

Temperatures and conditions for the coming **hours** are much more useful than for the coming **days**.

**Comment by str1k3r** [comment]
@ianmacd yeah, exactly. sorry, was not so clear.

**Comment by deno750** [comment]
@str1k3r @ianmacd Thanks for the feedbacks. We'll try to make a good hourly widget
